src/fund.jl

```diff
@@ -30,211 +34,229 @@ include("ImpactSeaLevelRiseComponent.jl")
-    connectparameter(m, :impactdeathmorbidity, :diasick, :impactdiarrhoea, :diasick)
-
-    connectparameter(m, :impactwaterresources, :population, :population, :population)
-    connectparameter(m, :impactwaterresources, :income, :socioeconomic, :income)
-    connectparameter(m, :impactwaterresources, :temp, :climateregional, :temp)
-
-    connectparameter(m, :impactsealevelrise, :population, :population, :population)
-    connectparameter(m, :impactsealevelrise, :income, :socioeconomic, :income)
-    connectparameter(m, :impactsealevelrise, :sea, :ocean, :sea)
-    connectparameter(m, :impactsealevelrise, :area, :geography, :area)
-
-    connectparameter(m, :impactaggregation, :income, :socioeconomic, :income)
-    connectparameter(m, :impactaggregation, :heating, :impactheating, :heating)
-    connectparameter(m, :impactaggregation, :cooling, :impactcooling, :cooling)
-    connectparameter(m, :impactaggregation, :agcost, :impactagriculture, :agcost)
-    connectparameter(m, :impactaggregation, :species, :impactbiodiversity, :species)
-    connectparameter(m, :impactaggregation, :water, :impactwaterresources, :water)
-    connectparameter(m, :impactaggregation, :hurrdam, :impacttropicalstorms, :hurrdam)
-    connectparameter(m, :impactaggregation, :extratropicalstormsdam, :impactextratropicalstorms, :extratropicalstormsdam)
-    connectparameter(m, :impactaggregation, :forests, :impactforests, :forests)
-    connectparameter(m, :impactaggregation, :drycost, :impactsealevelrise, :drycost)
-    connectparameter(m, :impactaggregation, :protcost, :impactsealevelrise, :protcost)
-    connectparameter(m, :impactaggregation, :entercost, :impactsealevelrise, :entercost)
-    connectparameter(m, :impactaggregation, :deadcost, :impactdeathmorbidity, :deadcost)
-    connectparameter(m, :impactaggregation, :morbcost, :impactdeathmorbidity, :morbcost)
-    connectparameter(m, :impactaggregation, :wetcost, :impactsealevelrise, :wetcost)
-    connectparameter(m, :impactaggregation, :leavecost, :impactsealevelrise, :leavecost)
-
-    return m
-end
-
-function getfund(;nsteps=1050, datadir=joinpath(dirname(@__FILE__), "..", "data"), params=nothing)
     # ---------------------------------------------
-    # Load parameters
+    # Set dimensions
     # ---------------------------------------------
-    if params==nothing
-        parameters = loadparameters(datadir)
-    else
-        parameters = params
-    end
+
+    set_dimension!(FUND, :time, collect(1950:1950+nsteps))
+    set_dimension!(FUND, :regions, ["USA", "CAN", "WEU", "JPK", "ANZ", "EEU", "FSU", "MDE", "CAM", "LAM", "SAS", "SEA", "CHI", "MAF", "SSA", "SIS"])
+    set_dimension!(FUND, :cpools, 1:5)
 
     # ---------------------------------------------
-    # Construct model
+    # Create components
     # ---------------------------------------------
-    m = constructfund(nsteps=nsteps)
+
+    add_comp!(FUND, scenariouncertainty, :scenariouncertainty)
```

Comment:
Do we have to use this form? Can't we just write ``add_comp!(FUND, scenariouncertainty)``, and then it automatically names it ``:scenariouncertainty``? Same for all the other ones.